src/client/extension.ts

```diff
@@ -77,10 +77,14 @@ function startREPLProc(context: ExtensionContext, config: settings.ISettings, pw
             }
             client.sendRequest(EvaluateRequestMessage, evaluateArgs)
 
-            await proc.showTerminalIfVisible();
+            await proc.showTerminalIfNotVisible();
         }
 
 
+      let cmdShowTerminal = commands.registerCommand("powershell.showTerminal", async () =>  proc.showTerminal());
+      let cmdHideTerminal = commands.registerCommand("powershell.hideTerminal",  async () => proc.hideTerminal());
```

Comment:
```suggestion
      let cmdHideTerminal = commands.registerCommand("powershell.hideTerminal",  proc.hideTerminal);
```
I think this can be simplified